apiary.apib

```diff
@@ -298,6 +298,11 @@ _This will only affect the version for those API calls and won't affect any othe
 
 We list all backwards-incompatible changes here. As described above, new additions and forwards-compatible changes don’t need a new API version and will not appear in this list.
 
+#### Beta
+
+- The property `status` in `invoices.list` and `invoices.info` now returns `due` and `matched` instead of `booked`.
+- The `filter` `status` property in the `invoices.list` request now only accepts an `array` of values.
```

Comment:
we can, but why you would do that? having to types to deal with brings complexity?

Reply:
It will not break backward compatibility if you accept both.

Reply:
It will anyway be a BC break on the split of the `booked` to `due` and `matched`, so breaking it here makes it more suitable and less complex IMO. The developers don't have to think about if I want to filter only by one status should I send it in the array or just a string?

Reply:
> The developers don't have to think about "if I want to filter only by one status should I send it in the array or just a string?".

But they don't have to think about that... they can just pass an array with 1 item.

The other way around: if you want to filter on more values, you will know it needs to be an array instead of a string...

So I would be fine with keeping both options for the sake of "simplicity for the simple use case", but no strong opinions about it.